package.json

```
@@ -40,6 +41,7 @@
         "gbimage-bridge": "^0.2.1",
         "i18next": "^21.6.5",
         "js-cookie": "^2.2.1",
+        "logs": "^1.5.10",
```

Hi @meenakshi-deriv , do we need this?
```suggestion
        "logs": "^1.5.10",
```

---

Hi @habib-deriv 
I didn't install this package.